src/main/java/com/odysseusinc/arachne/portal/model/BaseUser.java

```diff
@@ -180,7 +180,7 @@
     @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
     protected Set<DataNodeUser> dataNodeUsers = new HashSet<>();
 
-    @ManyToMany(targetEntity = Tenant.class, fetch = FetchType.LAZY)
+    @ManyToMany(targetEntity = Tenant.class, fetch = FetchType.EAGER)
```

Comment:
Why do we need this change? And how many tenants can be on production? in the case of thousands, that can be a not good idea to always get them from DB.

Reply:
The majority of the users will have just one. Some users can have up to several but it is definitely not thousands.